contracts/interfaces/IFollowModule.sol

```diff
@@ -14,6 +14,8 @@ interface IFollowModule {
      *
      * @param profileId The token ID of the profile to initialize this follow module for.
      * @param data Arbitrary data passed by the profile creator.
+     *
+     * @return bytes The encoded data to emit in the hub.
```

Comment:
dont think placing the data type of the returning value after `@return` is part of the standard.
Apart from that, I saw is not completely uniform across the code base.

Reply:
Inconsistencies solved in dd82346393db214caf49d91a1cadbaec235a0693!